state/state.go

```diff
@@ -1099,6 +1101,15 @@ func (st *State) AddApplication(args AddApplicationArgs) (_ *Application, err er
 		RelationCount: len(peers),
 		Life:          Alive,
 	}
+	if model.Type() == ModelTypeCAAS {
+		series, _ := series.HostSeries()
```

Comment:
Series here is not the series of the host machine of the controller. It's the series of the caas cloud where for now is "kubernetes".
Same with arch